### DISCUSSION TITLE
How can i make this work with SD3?

### DISCUSSION THREAD

**Discussion by elen07zz** [runew0lf/RuinedFooocus#140] [General]
I have downloaded stableDiffusion3SD3_sd3MediumInclT5XXL.safetensors   10gb version and it's not working:
Failed to load stableDiffusion3SD3_sd3MediumInclT5XXL.safetensors, loading default model instead

**Comment by ptits** [comment]
use 6GB model, not 10GB

**Reply by runew0lf** [reply]
this is the correct answer!

**Reply by ptits** [reply]
Works like a blast! Thanks
